docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,104 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+
+For this example, we will be using the `192.168.1.10` and `fe80::192:168:1:10` addresses for the following steps. You will need to substitute your own addresses in the following steps.
+
+Notice that the `IPv4` and `IPv6` addresses suffixes matches the number `xxx.168.1.10` and `xxxx::192:168:1:10` respectively, this is intentional to recognize the addresses were manually assigned in the `Raspberry Pi`.
+
+Worth to mention that the `IPv6` address that starts with `fe80` is the `link-local` address and it is used in this guide, the other addresses that start with `2806` are the global addresses and they are assigned by the router.
```

Comment:
This part should go into the section below (Grab your IPv4 and IPv6 address from your Raspberry Pi).

Reply:
Got you, updated the lines to the correct section.